Dockerfile

```diff
@@ -35,4 +34,4 @@ WORKDIR /data
 
 COPY --from=alpine:latest /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 ENTRYPOINT ["/cortex-tenant"]
-CMD ["-config", "$CONFIG_FILE"]
```

Comment:
Pretty sure this never worked. It's not supported in `CMD`. I figured (and of course I can change it) that it's easier to just override it with a mount or with a custom image.